tests/testthat/test-german_map.R

```diff
@@ -0,0 +1,23 @@
+test_that("returns an object of the expected class", {
+  plot <- german_map(financial)
+  expect_s3_class(plot, "ggplot")
+})
+
+test_that("returns correct risk category values colors", {
+  data <- tibble(
+    postcode = c(53773L, 53774L, 53775L),
+    xctr_risk_category = c("high", "medium", "low"),
+    benchmark = rep("all", 3)
+  )
+  expected_colors <- list(rgb(1, 0, 0), rgb(1, 0.5, 0), rgb(0, 1, 0))
```

Comment:
You may name the list elements to more clearly reflect what you expect

```
  expected_colors <- list(high = rgb(1, 0, 0), medium = rgb(1, 0.5, 0), low = rgb(0, 1, 0))
```

You may also name after the colours, but if you're not sure yet maybe avoid hard-coding colours that you're likely to change soon.